src/generate-cache/main.c

```diff
@@ -174,12 +173,12 @@ int main(int argc, char *arg[])
     else if((!strcmp(arg[k], "-m") || !strcmp(arg[k], "--max-mip")) && argc > k + 1)
     {
       k++;
-      max_mip = (dt_mipmap_size_t)MIN(MAX(atoi(arg[k]), 0), 7);
+      max_mip = (dt_mipmap_size_t)MIN(MAX(atoi(arg[k]), 0), 8);
```

Comment:
Instead of 0 and 8, why not sure DT_MIPMAP_0 and DT_MIPMAP_8 ?

Reply:
you're right of course. I've just keeped the previous notation, but using DT_MIPMPA8? is more clear ! done